test/helpers/with-server.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+const {URL} = require('url');
+const createTestServer = require('create-test-server');
+
+exports.withServer = async (t, run) => {
+	const s = await createTestServer();
+
+	s.host = (new URL(s.url)).hostname;
```

Comment:
@sindresorhus Shouldn't it be named `s.hostname`?

Reply:
Yes